tools/pagerduty-migrator/migrator/resources/integrations.py

```diff
@@ -52,7 +55,14 @@ def migrate_integration(integration: dict, escalation_policies: list[dict]) -> N
 def create_integration(
     name: str, integration_type: str, escalation_chain_id: str
 ) -> None:
-    payload = {"name": name, "type": integration_type, "team_id": None}
+
```

Comment:
Seems like `create_integration` should be left unchaged since we already have set the `oncall_type` field earlier:
```
integration["oncall_type"] = PAGERDUTY_TO_ONCALL_VENDOR_MAP.get(vendor_name, default=FALLBACK_INTEGRATION_TYPE)
```

So I propose to remove this and only keep
```
payload = {"name": name, "type": integration_type, "team_id": None}
```
as it was before.